active/RadioButtonsAPI.md

```diff
@@ -0,0 +1,153 @@
+# Background
+
+RadioButtons is our answer to enabling better keyboarding and narrator support for RadioButtons that need to be presented in a related list.
+
+Currently, the only way to group RadioButtons is through using a StackPanel and individually labeling each RadioButton through its GroupName property in order for narrator to read out the options somewhat understandably.
+
+However, using that method still left gaps in the narrator experience and StackPanel does not behave the way a user would expect when navigating the list via keyboard - RadioButtons was introduced to help solve those issues.
+
+# Description
+
+RadioButtons is a new control that enables you to create related lists of RadioButtons easily, while also fully supporting keyboarding and narrator functionality.
+
+# Examples
+
+The following example demonstrates how to create a small collective list of RadioButtons using RadioButtons.
+
+```xaml
+<RadioButtons Header="App Mode" SelectedIndex="2">
+    <x:String>Dark</x:String>
+    <x:String>Light</x:String>
+    <x:String>Windows Default</x:String>         
+</RadioButtons>
+```
+
+### DataBinding with RadioButtons
+An example of simple data binding of the ItemsSource with RadioButtons.
```

Comment:
The example below doesn't show data binding with _RadioButtons_ - it shows data binding to _RadioButtonItems_. Is this the expected usage? Or should it look like: 

```
        radioButtonItems = new List<RadioButton>();
        radioButtonItems.Add(new RadioButton() { Content = "Item 1" });
```

?

Reply:
I was confused too, RadioButtonItem isn't a UI type it's a custom data type defined above in the sample. But I think the sample type should be changed to like "MyOption" or something like that to reduce confusion.

Reply:
You can't put RadioButtons into the RadioButton**s** group because you'll get this:

![image](https://user-images.githubusercontent.com/6964369/53208807-0d68bd00-35ed-11e9-9dee-bc6ca6994b87.png)

Reply:
I think I fixed this by taking Jevan's suggestion. Take a look!